### ISSUE TITLE
Resuming training does not take into account the last used learning rate

### ISSUE THREAD

**Issue by lewfish** [azavea/raster-vision#4]
In https://github.com/azavea/keras-semantic-segmentation/blob/develop/src/model_training/train.py, we resume training from saved files by getting the last epoch number from `log.txt` and then use that as the `initial_epoch` argument to `fit_generators`. But that doesn't take into account the latest learning rate that was being used, which could have been set by the `ReduceLROnPlateau` callback.

**Comment by lewfish** [comment]
https://github.com/fchollet/keras/issues/1872 is relevant

**Comment by lewfish** [comment]
This issue is no longer applicable to the current and future versions of Raster Vision.
